packages/material/test/renderers/MaterialArrayControl.test.tsx

```diff
@@ -167,6 +167,41 @@ describe('Material array control', () => {
     expect(headerColumns).toHaveLength(3);
   });
 
+  it('should render even without properties', () => {
+    const store = initJsonFormsStore();
+    // re-init
+    const data: any = { test: [] };
+    const schema: JsonSchema = {
+      type: 'object',
+      properties: {
+        test: {
+          type: 'array',
+          items: { type: 'object' }
```

Comment:
shouldn't the items object contain a `properties` property instead of `type:'object'` ?

Reply:
The crash happens exactly in this case: Items are defined as `object` without defining further properties. If there was a `properties`key, the crash didn't happen.